src/black/linegen.py

```diff
@@ -1180,9 +1181,12 @@ def delimiter_split(
     except ValueError:
         raise CannotSplit("No delimiters found") from None
 
-    if delimiter_priority == DOT_PRIORITY:
-        if bt.delimiter_count_with_priority(delimiter_priority) == 1:
-            raise CannotSplit("Splitting a single attribute from its owner looks wrong")
+    if (
```

Comment:
We might want to keep this one as is, see the comment on the PR.

Reply:
Sorry, I'm not quite sure which COMMENT you're referring to?